examples/getting_started/pt/nvflare_lightning_getting_started.ipynb

```diff
@@ -0,0 +1,545 @@
+    "    - FL server sends the global model to clients and waits for their updates using `self.send_model_and_wait()`.\n",
+    "    - FL server aggregates all the `results` and produces a new global model using `self.update_model()`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d62a13d5-1130-44e6-8818-70e30de401e6",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "class FedAvg(BaseFedAvg):\n",
+    "    def run(self) -> None:\n",
+    "        self.info(\"Start FedAvg.\")\n",
+    "\n",
+    "        model = self.load_model()\n",
+    "        model.start_round = self.start_round\n",
+    "        model.total_rounds = self.num_rounds\n",
+    "\n",
+    "        for self.current_round in range(self.start_round, self.start_round + self.num_rounds):\n",
+    "            self.info(f\"Round {self.current_round} started.\")\n",
+    "            model.current_round = self.current_round\n",
+    "\n",
+    "            clients = self.sample_clients(self.min_clients)\n",
+    "\n",
+    "            results = self.send_model_and_wait(targets=clients, data=model)\n",
+    "\n",
+    "            aggregate_results = self.aggregate(results)\n",
+    "\n",
+    "            model = self.update_model(model, aggregate_results)\n",
+    "\n",
+    "            self.save_model(model)\n",
+    "\n",
+    "        self.info(\"Finished FedAvg.\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d24b6476-089a-4e9d-825b-07107bd5d84a",
+   "metadata": {},
+   "source": [
+    "### Client Code \n",
+    "Given a CIFAR10 [PyTorch Lightning](https://lightning.ai/) code example with the network wrapped into a LightningModule [LitNet](src/lit_net.py) class, we wish to adapt this centralized training code to something that can run in a federated setting."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c551053-5460-4d83-8578-796074170342",
+   "metadata": {},
+   "source": [
+    "On the client side, the training workflow is as follows:\n",
+    "1. Receive the model from the FL server.\n",
+    "2. Perform local training on the received global model\n",
+    "and/or evaluate the received global model for model\n",
+    "selection.\n",
+    "3. Send the new model back to the FL server."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c02bfc2a-783c-494f-9427-c38f40a2e870",
+   "metadata": {},
+   "source": [
+    "Using NVFlare's client API, we can easily adapt machine learning code that was written for centralized training and apply it in a federated scenario.\n",
+    "For a general use case, there are two essential methods to achieve this using the Client Lightning API :\n",
+    "- `flare.patch(trainer)`: Patch the lightning trainer. After flare.patch, the functions such as `trainer.fit()` and `trainer.validate()` will get the global model internally and automatically send the result model to the FL server.\n",
+    "- `receive()`: Receives model from the FL server."
```

Comment:
`receive()` is not used in lighting client api.

Reply:
Good catch, it is not required but can be used if wanted such as in
https://github.com/NVIDIA/NVFlare/blob/main/examples/getting_started/pt/src/cifar10_lightning_fl.py#L87
I'll clarify this in the notebook.